### PR TITLE
Adjust to the new channel identifier

### PR DESCRIPTION
**Pull request by palango**
I store the `channel_identifier` as `bytes` right now, but it might be nicer to switch to a `str`. Let me know what you think.

### CODE REVIEW

**Comment by pcppcp** [comment]
oh, yes, please, if it's a hash, make it a hexencoded string

File: raiden_libs/messages/balance_proof.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def __init__(
         locksroot: str = '0x%064x' % 0
     ) -> None:
         super().__init__()
-        assert channel_identifier > 0
+        assert isinstance(channel_identifier, T_ChannelIdentifier)
```
Comment (pcppcp) [inline raiden_libs/messages/balance_proof.py]: why not `ChannelIdentifier`?

**Comment by palango** [comment]
Updated to hexencoded strings

File: tests/test_client_mock.py [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
 from raiden_libs.utils.signing import eth_verify
 
 
 def test_client_multiple_topups(generate_raiden_clients):
     deposits = [1, 1, 2, 3, 5, 8, 13]
     c1, c2 = generate_raiden_clients(2)
-    channel_id = c1.open_channel(c2.address)
-    assert channel_id > 0
+    channel_identifier = c1.open_channel(c2.address)
+    assert is_bytes(channel_identifier)
```
Comment (pcppcp) [inline tests/test_client_mock.py]: this should be `is_instance(channel_identifier, str)` no? Or even better a custom function 

```
def is_channel_identifier(channel_id):
   return len(decode_hex(channel_id)) == 32
```

Comment (palango) [reply tests/test_client_mock.py]: Fixed

**Pull request merged**